andino_firmware/src/commands.h

```diff
@@ -69,19 +69,23 @@ namespace andino {
 /// @brief CLI commands.
 struct Commands {
   /// @brief Reads an analog GPIO.
-  static constexpr char kReadAnalogGpio{'a'};
+  static constexpr const char* kReadAnalogGpio{"a"};
   /// @brief Reads a digital GPIO.
-  static constexpr char kReadDigitalGpio{'d'};
+  static constexpr const char* kReadDigitalGpio{"d"};
   /// @brief Reads the encoders tick count values.
-  static constexpr char kReadEncoders{'e'};
+  static constexpr const char* kReadEncoders{"e"};
+  /// @brief Reads if there is an IMU connected.
+  static constexpr const char* kReadHasImu{"h"};
+  /// @brief Reads the encoders tick count values and IMU data.
```

Comment:
Could you please improve documentation by elaborating on what "IMU data" means? What items from the following list are expected to be retrieved: https://learn.adafruit.com/adafruit-bno055-absolute-orientation-sensor/overview#data-output-2974314?